workspaces/adventure-pack/goodies/typescript/Math.gcd/index.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
+
+describe("Math.gcd", () => {
+  it.each([
+    [48, 18, 6],
+    [17, 13, 1],
+    [36, 60, 12],
+    [5, 0, 5],
+    [0, 5, 5],
+    [-5, 10, 5],
+    [-7, 3, 1],
+    [-8, -12, 4],
+    [0, 0, 0],
+  ])("should return %p for gcd(%p, %p)", (a, b, expected) => {
```

Comment:
I believe the arguments get filled into the test case name in the same order as they are given, so right now it would read like `return a for gcd(b, expected)`.

```suggestion
  ])("Math.gcd(%p, %p) === %p", (a, b, expected) => {
```